maps/029-2.tmx

```diff
@@ -944,5 +944,189 @@
+   <properties>
+    <property name="callfunc" value="Sorfina"/>
+    <property name="sprite" value="154"/>
+    <property name="trigger" value="0,1"/>
+   </properties>
+  </object>
+  <object id="11" name="#Carpet" type="npc" x="1056" y="864" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="Sorfina_carpet"/>
+   </properties>
+  </object>
+  <object id="13" name="Dresser#tutorial" type="npc" x="928" y="736" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="Sorfina_dresser"/>
+    <property name="sprite" value="400"/>
+   </properties>
+  </object>
+  <object id="14" name="#DoorOutTut1" type="npc" x="1408" y="992" width="32" height="32">
+   <properties>
+    <property name="arguments" value="112,85"/>
+    <property name="callfunc" value="Sorfina_door"/>
+    <property name="sprite" value="45"/>
+    <property name="trigger" value="0,0"/>
+   </properties>
+  </object>
+  <object id="15" name="#SorfinaRules#1" type="npc" x="736" y="768" width="32" height="32">
+   <properties>
+    <property name="arguments" value="1"/>
+    <property name="callfunc" value="Sorfina_rules"/>
+    <property name="trigger" value="0,1"/>
+   </properties>
+  </object>
+  <object id="16" name="#SorfinaRules#2" type="npc" x="672" y="768" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="Sorfina_rules"/>
+    <property name="trigger" value="0,1"/>
+   </properties>
+  </object>
+  <object id="18" name="#SorfinaKnockBack#1" type="npc" x="704" y="832" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="Sorfina_knockback"/>
+    <property name="trigger" value="1,0"/>
+   </properties>
+  </object>
+  <object id="19" name="#SorfinaKnockBack#2" type="npc" x="768" y="768" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="Sorfina_knockback"/>
+    <property name="trigger" value="0,1"/>
+   </properties>
+  </object>
+  <object id="21" name="TutDebug" type="npc" x="672" y="832" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="TutorialDebug"/>
+    <property name="debug" value="true"/>
+    <property name="sprite" value="154"/>
+   </properties>
+  </object>
+  <object id="22" name="Debug#0" type="npc" x="960" y="832" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="Debug"/>
+    <property name="debug" value="true"/>
+    <property name="sprite" value="154"/>
+   </properties>
+  </object>
+  <object id="24" name="#CandorBank#Door" type="npc" x="1120" y="3904" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="CandorBank_Door"/>
+    <property name="sprite" value="45"/>
+    <property name="trigger" value="0,0"/>
+   </properties>
+  </object>
+  <object id="25" name="Barrel#0" type="npc" x="736" y="2752" width="32" height="32">
+   <properties>
+    <property name="arguments" value="0"/>
```

Comment:
@wushin the parser should be able to tell the difference between lack of arguments and 0